doc/conf.py

```diff
@@ -93,7 +93,7 @@
 
 # -- PDF support via http://code.google.com/p/rst2pdf/
 try:
-    import rst2pdf
+    import rst2pdf  # noqa
     extensions.append('rst2pdf.pdfbuilder')
```

Comment:
Is `#noqa` a flake8 hint? We've been using `pylint` (see `make pylint`), but have not utilized it extensively.

Reply:
Yes, flake8 will ignore errors on any lines marked with this comment. I'll rebase this and mark the other imports that should be kept if you want.

Reply:
We'd need to decide to swtich to `flake8`. Generally we have avoided adding hints like that to the codebase, but I don't think we'd have a problem with it if we formally adopted the tool.

Presently we get a huge amount of output from `pylint` and it makes it rather useless (`make pylint`). We probably need to do some tuning to either `flake8` or `pylint` to match our [CodingStyle](https://trac.edgewall.org/wiki/TracDev/CodingStyle).